docs/DeveloperGuide.md

```diff
@@ -416,18 +416,23 @@ There are currently 4 custom predicates implemented in FAST:
 `TagMatchesKeywordPredicate` implemented by running the tags through a for-loop and checking if any of them match the given keyword.
 `RemarkContainsKeywordPredicate` implemented by using the inbuilt `String::contains`.
 
+![Find_Command_Class_Diagram](images/findcommandpredicates.png)
+
 Given below is an example usage scenario and how the find mechanism behaves at each step.
 
 Step 1. The user launches the application for the first time. <br>
 Step 2. The user inputs `find john` in the CLI to find all contacts whose names contain `john`. This calls `LogicManager::execute` which in turn
 calls `FastParser::parseCommand` to parse the given input. <br>
 Step 3. `FastParser` will determine that it is a find command and will call `FindCommandParser::parse`. From the given input,
-`FindCommandParser` will determine that the user is searching for a name and return a `FindCommand` with a `NameContainsQueriesPredicate`. <br>
+`FindCommandParser` will determine that the user is searching for a name and return a `FindCommand` with a `NameContainsQueriesPredicate` 
+containing a `List` of all the search queries (only "john" in this case) <br>
 Step 4. After execution of the user input, `LogicManager` calls `FindCommand::execute(model)` where model contains methods that mutate
 the state of our contacts. <br>
 Step 5. Through a series of method chains, it calls `ModelManager::getFilteredPersonList()`, which will display the results
 of the search.<br>
 
+![Find_Command_Sequence_Diagram](images/findcommandsequencediagram.png)
```

Comment:
Your sequence diagram does not have any activation bar, which is required for sequence diagrams. You can add it in the puml file with `activate [COMPONENT]`

Reply:
I think for the sequence diagram, it would be good to use the color scheme that is used throughout the DG. In this case, logic can be blue color.